### ISSUE TITLE
Error installing node-sass

### ISSUE THREAD

**Issue by tschf** [OraOpenSource/apex-frontend-boost#79]
I tried this a while back, and get node version errors. Since 16.04 of Ubuntu was not far off, I thought I'd wait to see in case the version met the minimum requirements.

As per: http://packages.ubuntu.com/xenial/nodejs, the version of node available in the repos is 4.2.6

Final output:
```
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.11
npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! node-sass@3.5.3 install: `node scripts/install.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the node-sass@3.5.3 install script 'node scripts/install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-sass
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/trent/Projects/apex-frontend-boost/npm-debug.log
```

and tail of `npm-debug.log` where the errors begin

```
69438 error Linux 4.4.0-21-generic
69439 error argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
69440 error node v4.2.6
69441 error npm  v3.5.2
69442 error file sh
69443 error code ELIFECYCLE
69444 error errno ENOENT
69445 error syscall spawn
69446 error node-sass@3.5.3 install: `node scripts/install.js`
69446 error spawn ENOENT
69447 error Failed at the node-sass@3.5.3 install script 'node scripts/install.js'.
69447 error Make sure you have the latest version of node.js and npm installed.
69447 error If you do, this is most likely a problem with the node-sass package,
69447 error not with npm itself.
69447 error Tell the author that this fails on your system:
69447 error     node scripts/install.js
69447 error You can get information on how to open an issue for this project with:
69447 error     npm bugs node-sass
69447 error Or if that isn't available, you can get their info via:
69447 error     npm owner ls node-sass
69447 error There is likely additional logging output above.
69448 verbose exit [ 1, true ]
```

**Comment by tschf** [comment]
I noticed when trying to install `node-sass` standalone, it was trying to call `node` rather than `nodejs` to run `script/install.js`.

```
npm install node-sass
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm

> node-sass@3.5.3 install /tmp/node_modules/node-sass
> node scripts/install.js

sh: 1: node: not found
```

The nodejs package doesn't include a `node` file.

```
dpkg -L nodejs | grep "bin/node"
/usr/bin/nodejs
```

I resolved this by adding a symbolic link to `node`

```
cd /usr/bin
sudo ln -s nodejs node
```

**Comment by vincentmorneau** [comment]
Hi Trent, thanks for looking into this.

What would you suggest we add to the docs exactly? I don't quite understand if you've resolved it.

Thanks.

**Comment by tschf** [comment]
Hi.. 
tl;dr - yes the issue is resolved

When I first got the error, I thought it way maybe due to an older version of node, but after looking deeper I found it because of the command `node` not resolving to anything - which `node-sass` was seemingly using for the installation. On Ubuntu, the command for node is `nodejs` and not `node`.

Even on the [`node-sass`](https://www.npmjs.com/package/node-sass) page, it has a brief mention of this:

> Some users have reported issues installing on Ubuntu due to node being registered to another package. Follow the official NodeJS docs to install NodeJS so that #!/usr/bin/env node correctly resolved.

See also this: http://stackoverflow.com/questions/18130164/nodejs-vs-node-on-ubuntu-12-04

I'm not sure if it's worth adding to the docs or not?
